screens/TrackerScreen.js

```diff
@@ -30,18 +30,28 @@ const TrackerScreen = () => {
   const animationRef = useRef(null);
   const [loading, setLoading] = React.useState(false);
 
-  const amountUsed = 6200;
-  const budgetAmount = 10000;
-  const budgetLeft = budgetAmount - amountUsed;
-  const percentageUsed = (amountUsed / budgetAmount) * 100;
-
-  const data = {
-    labels: ["Jan", "Feb", "Mar", "Apr", "May", "Jun"],
-    datasets: [
-      {
-        data: [6813, 5200, 2000, 7240, 5568, 9000],
-      },
+  // Todo: Add state setter to update from default request data to new
+  const requestData = {
```

Comment:
Add a state setter variable, that will simulate the calling of api. Add in other screens as well.